doc/man7/EVP_PKEY-ML-DSA.pod

```diff
@@ -44,6 +44,40 @@ and private key output to B<PKCS#8> files will by default include the seed.
 When the seed was not initially known, or was not retained, B<PKCS#8> private
 key files will contain only the private key in FIPS 204 C<sk> format.
 
+=item "seed-only" (B<OSSL_PKEY_PARAM_SEED_ONLY>) <unsigned int>
+
+When this parameter is set, its value is ignored, and the generated key is
```

Comment:
As a general style approach, I would not have an ignored value here. I would require the user to provide a fixed value (e.g. 1) simply to be consistent and allow for any future possible choices where we may want to expand the values for this parameter. 

If users are free to put whatever they like then we can *never* use the value. 
Fixing it to 1 allows for possible future changes - which we may never make.

Reply:
A boolean setting makes little sense though, because just not setting the parameter is the correct/simplest way to not force seed-only. Setting it to zero meaning "do nothing" makes no sense.

If we want to support more values, they should have some non-default effect. For example:

0. Do not output the seed, giving an expanded key-only output.
1. As in this PR, output only the seed.
2. Output both seed and expanded key

All three cases override the provparam output-format, if different.

Basically, any setting here needs to mean something concrete, rather than just be a NOP.